docs/modules/ROOT/pages/api/finance.adoc

```diff
@@ -208,11 +208,13 @@ Returns the total vested amount of a specified `token` at a given `timestamp`.
 
 Releases the amount of a given `token` that has already vested and returns that amount.
 
+NOTE: If the releasable amount is zero, this function won't emit the event.
+
```

Comment:
Should we mention that `ERC20::transfer` won't be attempted when amt == 0 as well?